http facade for the sqlite website is no longer operational

Trying to download `sqlite` from its home no longer works. All simply because the `http://www.sqlite.org/` website seems to be down. Meanwhile the `https://www.sqlite.org/` one seems to be operational.

### Setup
_Package Name:_ `cross/spk`
_Package Version:_ Any

### Expected behavior
`make digests` should work.

### Actual behavior
`make digests` fails with connection errors

### Steps to reproduce
_1._ Ensure that the `distrib` folder is purged of `sqlite-autoconf-` resources
_2._ Run `make digests` in `cross/spk`

#### Other logs
```
===>  Downloading files for sqlite-autoconf
wget http://www.sqlite.org/2018/sqlite-autoconf-3250300.tar.gz
échec : Connection refused.
échec : Connection refused.
make: *** [download_target] Error 4
```

--- a/cross/sqlite/Makefile
+++ b/cross/sqlite/Makefile
@@ -2,14 +2,14 @@ PKG_NAME = sqlite-autoconf
 PKG_VERS = 3250300
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://www.sqlite.org/2018
+PKG_DIST_SITE = https://www.sqlite.org/2018
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
 DEPENDS = cross/zlib cross/speex
 
-HOMEPAGE = http://www.sqlite.org/
+HOMEPAGE = https://www.sqlite.org/
 COMMENT  = Self-contained, serverless, zero-configuration, transactional SQL database engine.
-LICENSE  = http://www.sqlite.org/copyright.html
+LICENSE  = https://www.sqlite.org/copyright.html
 
 GNU_CONFIGURE = 1
 

